lib/src/async_queue.dart

```diff
@@ -59,12 +59,13 @@ class AsyncQueue<T> {
   /// the process was cancelled.
   Future _processNextItem() {
     var item = _items.removeFirst();
-    return _processor(item).then((_) {
+    return _processor(item).then((_) async {
       if (_items.isNotEmpty) return _processNextItem();
 
       // We have drained the queue, stop processing and wait until something
       // has been enqueued.
       _isProcessing = false;
+      return await null;
```

Comment:
I meant the other `return` should be `return await`. There's no use in awaiting `null`.

Reply:
Missed that, fixed.